tgui/packages/tgui/interfaces/Reagents.js

```diff
@@ -1,9 +1,71 @@
 import { useBackend, useLocalState } from '../backend';
 import { Box, Button, LabeledList, Section, Table, Icon, Chart, Flex, Stack } from '../components';
-import { TableCell, TableRow } from '../components/Table';
 import { Window } from '../layouts';
 import { map } from 'common/collections';
 
+const BRUTE = 1 << 0;
+const BURN = 1 << 1;
+const TOXIN = 1 << 2;
+const OXY = 1 << 3;
+const CLONE = 1 << 4;
+const HEALING = 1 << 5;
+const DAMAGING = 1 << 6;
+const EXPLOSIVE = 1 << 7;
+const OTHER = 1 << 8;
+const DANGEROUS = 1 << 9;
+const EASY = 1 << 10;
+const MODERATE = 1 << 11;
+const HARD = 1 << 12;
+const ORGAN = 1 << 13;
+const DRINK = 1 << 14;
+const FOOD = 1 << 15;
+const SLIME = 1 << 16;
+const DRUG = 1 << 17;
+const UNIQUE = 1 << 18;
+const CHEMICAL = 1 << 19;
+const PLANT = 1 << 20;
+const COMPETITIVE = 1 << 21;
```

Comment:
These should be passed down through `ui_static_data` so we don't have two places to update if we want to change the flags.

Reply:
true

Reply:
Hmm, do I still set them up as consts then?

Reply:
return them via ui_static_data